controllers/api/menu.controller.js

```diff
@@ -1,23 +1,63 @@
-﻿var config = require('config.json');
-var express = require('express');
+﻿var express = require('express');
 var router = express.Router();
-var menuService = require('services/menu.service');
```

Comment:
so you did all of it here and not use the menu.service for queries...
then, ok, we will change it later.